HLTrigger/Configuration/python/customizer_AlpakaPixelTracking_Phase2_141X.py

```diff
@@ -0,0 +1,247 @@
+# adapted from  https://github.com/cms-sw/cmssw/blob/CMSSW_14_0_0/HLTrigger/Configuration/python/customizeHLTforAlpaka.py#L579
+import FWCore.ParameterSet.Config as cms
+from HeterogeneousCore.AlpakaCore.functions import *
+
+def AlpakaHLTPixelRecoLocal(process):
+    '''Customisation to introduce the Local Pixel Reconstruction in Alpaka.
+    '''
+    process.hltESPSiPixelCablingSoA = cms.ESProducer('SiPixelCablingSoAESProducer@alpaka', 
+            CablingMapLabel = cms.string(''),
+            UseQualityInfo = cms.bool(False),
+            appendToDataLabel = cms.string(''),
+            alpaka = cms.untracked.PSet(
+                backend = cms.untracked.string('')
+        )
+    )
+    process.hltESPSiPixelGainCalibrationForHLTSoA = cms.ESProducer('SiPixelGainCalibrationForHLTSoAESProducer@alpaka',
+            appendToDataLabel = cms.string(''),
+            alpaka = cms.untracked.PSet(
+                backend = cms.untracked.string('')
+        )
+    )
+    process.hltESPPixelCPEFastParamsPhase2 = cms.ESProducer('PixelCPEFastParamsESProducerAlpakaPhase2@alpaka', 
+            ComponentName = cms.string("PixelCPEFastParamsPhase2"),
+            appendToDataLabel = cms.string(''),
+            alpaka = cms.untracked.PSet(
+                backend = cms.untracked.string('')
+        )
+    )
+    process.hltOnlineBeamSpotDevice = cms.EDProducer('BeamSpotDeviceProducer@alpaka',
+            src = cms.InputTag('hltOnlineBeamSpot'),
+            alpaka = cms.untracked.PSet(
+                backend = cms.untracked.string('')
+        )
+    )
+    process.siPixelClustersSoA = cms.EDProducer('SiPixelPhase2DigiToCluster@alpaka',
```

Comment:
Modules run at HLT start their label with ```hlt``` - this should now also be done for Phase-2.

Reply:
I changed the name only for the new collections added by the customiser in beb3cb2d8258f29fc2fd33ab4c12b11964e00872
There are several modules and sequences in the phase2 menu which do not follow the naming convention. This problem is known, it will be fixed in a future PR